### PR TITLE
feat: fix node ephemeral storage

### PR DESCRIPTION
**Pull request by alexeldeib**
follow on to #413 + RP changes to fix node ephemeral storage and expose it in RP API

### CODE REVIEW

File: vhdbuilder/packer/vhd-image-builder-gen2.json [hunk 1]
```diff
@@ -225,6 +225,16 @@
       "source": "parts/linux/cloud-init/artifacts/nvidia-device-plugin.service",
       "destination": "/home/packer/nvidia-device-plugin.service"
     },
+    {
```
Comment (eriksywu) [inline vhdbuilder/packer/vhd-image-builder-gen2.json:228]: Hmm, is the intention for the the bind-mount unit files be present in the VHD? I see that you're also injecting it via nodecustomdata.

Comment (alexeldeib) [reply vhdbuilder/packer/vhd-image-builder-gen2.json:228]: yeah actually idk why I did both...

Comment (alexeldeib) [reply vhdbuilder/packer/vhd-image-builder-gen2.json:228]: probably fine in just custom data

File: parts/linux/cloud-init/artifacts/bind-mount.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+MOUNT_POINT="/mnt/aks"
+{{end}}
+
+KUBELET_MOUNT_POINT="${MOUNT_POINT}/kubelet"
+KUBELET_DIR="/var/lib/kubelet"
+
+mkdir -p "${MOUNT_POINT}"
+
+# only move the kubelet directory to alternate location on first boot.
+SENTINEL_FILE="/opt/azure/containers/bind-sentinel"
```
Comment (alexeldeib) [inline parts/linux/cloud-init/artifacts/bind-mount.sh:27]: can probably just test if "${MOUNT_POINT}" exists

**Comment by mainred** [comment]
Please ignore AKS Windows VHD Pipeline check, I am fixing this. Sorry for the trouble.

**Pull request closed without merging**